awx_collection/plugins/modules/tower_group.py

```diff
@@ -112,9 +112,21 @@ def main():
     description = module.params.get('description')
     state = module.params.pop('state')
     variables = module.params.get('variables')
+    hosts = module.params.get('hosts')
+    children = module.params.get('children')
 
     # Attempt to look up the related items the user specified (these will fail the module if not found)
     inventory_id = module.resolve_name_to_id('inventories', inventory)
+    hosts_ids = None
+    if hosts is not None:
+        hosts_ids = []
+        for item in hosts:
+            hosts_ids.append(module.resolve_name_to_id('hosts', item))
+    children_ids = None
+    if children is not None:
+        children_ids = []
+        for item in children:
+            children_ids.append(module.resolve_name_to_id('groups', item))
```

Comment:
this directly conflicts with `association_fields`, I think this would be better split out into another PR that just changes the field name, which I agree should be done.

Reply:
Agreed, this PR should have just been for tower_job_template. @beeankha let me know if you want to pull this out of here or if I should?

Reply:
I can do this, and put the changes into a new `tower_group.py`-specific PR.

Reply:
resolved separately